project/MiMa.scala

```diff
@@ -0,0 +1,81 @@
+// It would be nice to use sbt-mima-plugin here, but the plugin is missing
+// at least two features we need:
```

Comment:
It would be nice to feed this back to MiMa so we can rid of the duplication in the future.